mmcv/model_zoo/hub.py

```diff
@@ -0,0 +1,117 @@
+import subprocess
+from typing import Union
+
+import torch
+
+from mmcv import dump, load
+from mmcv.utils import Config
+
+
+def update_model_zoo(name: str, config: str, details: dict,
```

Comment:
Maybe we have to add information about the name of the repo in JSON, we need this information to get the corresponding builder.